lnbits/core/views/api.py

```diff
@@ -580,3 +583,23 @@ async def api_fiat_as_sats(data: ConversionData):
         output["sats"] = await fiat_amount_as_satoshis(data.amount, data.from_)
         output["BTC"] = output["sats"] / 100000000
         return output
+
+@core_app.get("/api/v1/qrcode/{data}", response_class=StreamingResponse)
+async def img(request: Request, data):
+    qr = pyqrcode.create(data)
+    stream = BytesIO()
+    qr.svg(stream, scale=3)
+    stream.seek(0)
+
+    async def _generator(stream: BytesIO):
+        yield stream.getvalue()
+
+    return StreamingResponse(
+        _generator(stream),
+        headers={
+            "Content-Type": "image/svg+xml",
+            "Cache-Control": "no-cache, no-store, must-revalidate",
+            "Pragma": "no-cache",
+            "Expires": "0",
+        },
+    )
```

Comment:
Looks good to me. In the long run we might have to rate limit this call. 

Or make it a paid service :thinking:

Reply:
Yeah, I was thinking that, could be great. Can I rate limit a specific endpoint per ip in fastapi?

Reply:
There are packages that add rate limiting capabilities to routes. As it is an additional dependency I'd only introduce this if we see abuse in future and we absolutely need to.

slowapi: https://github.com/laurentS/slowapi